setup.cfg

```diff
@@ -10,16 +10,28 @@ author_email = guettliml.django-check-html-middleware@thomas-guettler.de
 classifiers =
     Environment :: Web Environment
     Framework :: Django
+    Framework :: Django :: 2.2
+    Framework :: Django :: 3.0
+    Framework :: Django :: 3.1
+    Framework :: Django :: 3.2
+    Framework :: Django :: 4.0
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
Is this list needed? I guess nobody uses a 3.5 or lower. And nevertheless some lines later there is >=3.6.

The rest looks good.  I will merge it after we agreed on above topic. Thank you.

Reply:
It will be shown at PyPI. 
Here is an [example how it looks at PyPI](https://pypi.org/project/djangorestframework/)

Reply:
Who will update it, when Python 3.11 will be released? I am super lazy. I don't want to do it. Please tell me why the explicit versions are needed. If they are not needed, then please update the PR.